### ISSUE TITLE
about your code

### ISSUE THREAD

**Issue by denghuiru** [mks0601/Photo-Realistic-Single-Image-Super-Resolution-Using-a-Generative-Adversarial-Network#1]
Hi, 
As you write, you have implemented the SRGAN network, however, I am wonder if you will release your implementation code recently.

**Comment by mks0601** [comment]
there is a SRGAN branch

**Comment by denghuiru** [comment]
@mks0601 
hi, i have found the corresponding code of SRResNet and SRGAN, do you have some instructions about how to execute your code. 
Thanks!

**Comment by denghuiru** [comment]
@mks0601 
hi, I have a small question about the training data preparation, does the training data must generated by the generate_train.m, I just use some original HR images and its LR images, and the results is poor.

**Comment by mks0601** [comment]
yes. the code in dataGen folder generate training data and test data to usable form.

**Comment by denghuiru** [comment]
@mks0601 
hi, I found a problem in the training stage, you used image.load() to load the training image and GT image, however, the generate_train.m generate the training images to bin format, how it can matches to train the network!

**Comment by mks0601** [comment]
it`s been long time ago i implemented this paper, so i was confused.
maybe for training, you have to use just image following my code
and for test stage, you have to convert image files to bin files

all you have to do is just understand code and follow it.

**Comment by denghuiru** [comment]
@mks0601 
hi，I am wonder if you also use ImageNet in the training stage and how many images do you need? Because ImageNet is too large, so how many pictures are enough?

**Comment by mks0601** [comment]
i used imageNet and 50,000images are used

**Comment by denghuiru** [comment]
@mks0601 
hi, sorry to disturb you again. Recently, I have used imageNet (50,000) to train the SRResNet, the LR images are generated using Matlab bicubic, then,I have trained the nets with  the processed data for 750,000 iters, the PSNR is still 30.32dB on Set5, the paper is 31.92dB. So, I wonder if there are some points that I have not noticed.

**Comment by mks0601** [comment]
Did you check the PSNR of the image from MATLAB? Is it the same with the PSNR of the 'bicubic' of the paper?

**Comment by denghuiru** [comment]
@mks0601 
I have used Set5 and Set14 and BSD100 datasets to test the PSNR under matlab, the downsampling method is bicubic too, and the PSNR can match with the paper proposed. 
So I am wonder how you generate the LR images of ImageNet. Besides, whether need carefully choose the random sample from ImageNet, currently, I just choose the first 50,000 images from the ImageNet, not so randomly.

**Comment by mks0601** [comment]
I choose randomly from imagenet 2015 set
